apps/src/authoredHintUtils.js

```diff
@@ -104,6 +105,13 @@ authoredHintUtils.getFinishedHints_ = function () {
 };
 
 /**
+ * @return {FinishedHint[]}
+ */
+authoredHintUtils.getOldFinishedHints = function () {
```

Comment:
One nit, this will be correct for hints they've viewed this session but won't include any hints from previous sessions.  Totally fine for the in-classroom test which only needs to work for completing everything in a single session, but maybe worth calling out so we don't forget.